components/ActivityCards.js

```diff
@@ -45,6 +46,12 @@ export default function ActivityCards({ activities, editTag }) {
                   </Link>
                 )}
               </li>
+              <li>
+                {" "}
+                <button>
```

Comment:
And you use a button element or a link element. The button element is not necessary. You can style your link element like a button if it the appearance of a button that you want to have.